core/code/portal_data.js

```diff
@@ -143,46 +103,3 @@ window.findPortalLatLng = function(guid) {
 })();
 
 
-// get the AP gains from a portal, based only on the brief summary data from portals, links and fields
-// not entirely accurate - but available for all portals on the screen
-window.getPortalApGain = function(guid) {
```

Comment:
Ok, this is what you should do:
- put the method back
- add note to changelog that this method is deprecated
- add note to documentation (if present) about the depreciation
- open pr for after release where the method will be removed

That way we atleast give time developers who use this function in their plugins to make an update.